<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_Bancos</class>
 <widget class="QWidget" name="frm_Bancos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2c2c2c;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_9" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item alignment="Qt::AlignHCenter|Qt::AlignTop">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">QLabel {
    font-size: 32px;
    color: #FFFFFF;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
              </property>
              <property name="text">
               <string>DESPESAS</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_10" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QTableWidget" name="tableGastos">
              <property name="styleSheet">
               <string notr="true">QTableWidget, QTableView {
    border: 1px solid #dcdcdc; 
    border-radius: 5px; 
    gridline-color: #dcdcdc; 
    background-color: #ffffff; 
    font-size: 14px; 
    color: #333333; 
}

QHeaderView::section {
    background-color: #f0f0f0; 
    color: #333333;
    font-weight: bold; 
    border: 1px solid #dcdcdc; 
    padding: 4px; 
}

QTableWidget::item:selected, QTableView::item:selected {
    background-color: #b3d9ff; 
    color: #000000;
}

QTableCornerButton::section {
    background-color: #f0f0f0; 
    border: 1px solid #dcdcdc;
}

QScrollBar:vertical {
    background: #f0f0f0;
    width: 12px; 
    margin: 2px 0 2px 0; 
    border: none; 
}

QScrollBar::handle:vertical {
    background: #b0bec5; 
    min-height: 20px; 
    border-radius: 6px; 
}

QScrollBar::handle:vertical:hover {
    background: #90a4ae; 
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    background: none; 
    height: 0px; 
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none; 
}

QScrollBar:horizontal {
    background: #f0f0f0; 
    height: 12px; 
    margin: 0 2px 0 2px; 
    border: none;
}

QScrollBar::handle:horizontal {
    background: #b0bec5;
    min-width: 20px; 
    border-radius: 6px;
}

QScrollBar::handle:horizontal:hover {
    background: #90a4ae; 
}

QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    background: none;
    width: 0px;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}
</string>
              </property>
              <column>
               <property name="text">
                <string>Nome</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Fornecedor</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Observação</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Quantidade</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Valor</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_7" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_17">
         <item>
          <widget class="QWidget" name="widget_20" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QWidget" name="widget_21" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_18">
               <item alignment="Qt::AlignHCenter|Qt::AlignTop">
                <widget class="QLabel" name="label_4">
                 <property name="styleSheet">
                  <string notr="true">QLabel {
    font-size: 32px;
    color: #FFFFFF;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
                 </property>
                 <property name="text">
                  <string>BANCOS</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_22" native="true">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_19"/>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_19" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_16">
            <item alignment="Qt::AlignTop">
             <widget class="QPushButton" name="btn_consultaD">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 2px solid #d1c4b2;
    border-radius: 12px;
    color: #000000;
    font-size: 14px;
    font-weight: bold;
    padding: 10px 16px;
    background-image:url(:/icon_consulta/consulta.png); 
    background-repeat: no-repeat;
    background-position: center;
    transition: all 0.3s ease;
}

QPushButton:hover {
    background-color: #f2ebe0;
    border-color: #c4b49e;
}

QPushButton:pressed {
    background-color: #e8dfcf; 
    border-color: #b39b8d; 
    padding-left: 12px; 
    padding-top: 4px;
}</string>
              </property>
              <property name="text">
               <string>ㅤㅤ
ㅤㅤ
</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_11" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item alignment="Qt::AlignHCenter|Qt::AlignTop">
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">QLabel {
    font-size: 32px;
    color: #FFFFFF;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
              </property>
              <property name="text">
               <string>RENDIMENTOS</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_12" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QTableWidget" name="tableLucros">
              <property name="styleSheet">
               <string notr="true">QTableWidget, QTableView {
    border: 1px solid #dcdcdc; 
    border-radius: 5px; 
    gridline-color: #dcdcdc; 
    background-color: #ffffff; 
    font-size: 14px; 
    color: #333333; 
}

QHeaderView::section {
    background-color: #f0f0f0; 
    color: #333333;
    font-weight: bold; 
    border: 1px solid #dcdcdc; 
    padding: 4px; 
}

QTableWidget::item:selected, QTableView::item:selected {
    background-color: #b3d9ff; 
    color: #000000;
}

QTableCornerButton::section {
    background-color: #f0f0f0; 
    border: 1px solid #dcdcdc;
}

QScrollBar:vertical {
    background: #f0f0f0;
    width: 12px; 
    margin: 2px 0 2px 0; 
    border: none; 
}

QScrollBar::handle:vertical {
    background: #b0bec5; 
    min-height: 20px; 
    border-radius: 6px; 
}

QScrollBar::handle:vertical:hover {
    background: #90a4ae; 
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    background: none; 
    height: 0px; 
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none; 
}

QScrollBar:horizontal {
    background: #f0f0f0; 
    height: 12px; 
    margin: 0 2px 0 2px; 
    border: none;
}

QScrollBar::handle:horizontal {
    background: #b0bec5;
    min-width: 20px; 
    border-radius: 6px;
}

QScrollBar::handle:horizontal:hover {
    background: #90a4ae; 
}

QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    background: none;
    width: 0px;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}
</string>
              </property>
              <column>
               <property name="text">
                <string>Nome</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Fornecedor</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Observação</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Quantidade</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Valor</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_14" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
             <widget class="QLabel" name="label_SomaGastos">
              <property name="styleSheet">
               <string notr="true">QLabel {
    font-size: 32px;
    color: #FFFFFF;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
              </property>
              <property name="text">
               <string>Soma dos gastos:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_13" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QLineEdit" name="txt_gastos">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
	color: #000000;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_16" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
             <widget class="QLabel" name="label_SomaGastos_3">
              <property name="styleSheet">
               <string notr="true">QLabel {
    font-size: 32px;
    color: #FFFFFF;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
              </property>
              <property name="text">
               <string>Soma dos gastos:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_15" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <widget class="QLineEdit" name="txt_total">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
	color: #000000;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignBottom">
          <widget class="QWidget" name="widget_18" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QLabel" name="label_SomaGastos_2">
              <property name="styleSheet">
               <string notr="true">QLabel {
    font-size: 32px;
    color: #FFFFFF;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
              </property>
              <property name="text">
               <string>Soma dos gastos:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_17" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <item>
             <widget class="QLineEdit" name="txt_rendimentos">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
	color: #000000;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Img/icon_consultadiaria.qrc"/>
  <include location="../Img/icon_logoBancos.qrc"/>
 </resources>
 <connections/>
</ui>
